controls/cf_serverd.cf

```diff
@@ -32,9 +32,6 @@ body server control
       # Suggested value >= total number of hosts
       maxconnections        => "$(def.control_server_maxconnections)";
 
-      # Bind to all interfaces including ipv6
```

Comment:
@nickanderson Did you see the whole context? This is inside `!windows` context. So should apply just like before on everything except windows.